dev/test-zeekoe.py

```diff
@@ -263,7 +263,7 @@ def run_command_list(self, command_list):
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("command", help="", nargs="?", default="list")
-    parser.add_argument("--path", help="path to create configs", default="./dev")
+    parser.add_argument("--path", help="path to create configs", default=".")
```

Comment:
Non-blocking: While you're touching this code, could this be something a bit clearer like `--config-path`?